lib/publishing.js

```diff
@@ -61,6 +61,10 @@ function createDefaultPublisher() {
       return format('%s|%s', concretePublish(unionNode.left),
                     concretePublish(unionNode.right));
     },
+    INTERSECTION (unionNode, concretePublish) {
```

Comment:
I would prefer to format both intersections and unions with whitespace.

Reply:
Agreed. Added!